superset/views/core.py

```diff
@@ -507,6 +507,7 @@ class SliceAddView(SliceModelView):  # noqa
     list_columns = [
         'id', 'slice_name', 'slice_link', 'viz_type',
         'datasource_link', 'owners', 'modified', 'changed_on']
+    show_columns = SliceModelView.edit_columns + list_columns
```

Comment:
This seems prone to have duplicated columns, is this not an issue?

Reply:
It didn't cause an issue for me, but I agree it would be a good idea to dedupe. Fixed.